src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
filter icon not centered in data menu 
![image](https://user-images.githubusercontent.com/31045223/221855150-027b1e93-8402-42b9-ba86-139aeedf8279.png)

Reply:
yes, still some work to do on the size and dimension of the SVGs